### PR TITLE
[DOCS] Update control structures (2).

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/control-structures.rst [hunk 1]
```diff
@@ -398,17 +398,17 @@ If used properly, analysis tools can evaluate your contract to identify the cond
...
-The deprecated keyword ``throw`` can also be used as an alternative to ``revert()`` (but only without error message).
 
 .. note::
-    From version 0.4.13 the ``throw`` keyword is deprecated and will be phased out in the future.
+    There used to be a keyword called ``throw`` with the same semantics as ``revert()`` which
+    whas deprecated in version 0.4.13 and removed in version 0.5.0.
 
 When exceptions happen in a sub-call, they "bubble up" (i.e. exceptions are rethrown) automatically. Exceptions to this rule are ``send``
-and the low-level functions ``call``, ``delegatecall``, ``callcode`` and ``staticcall`` -- those return ``false`` in case
+and the low-level functions ``call``, ``delegatecall`` and ``staticcall`` -- those return ``false`` in case
```
Comment (leonardoalt) [inline docs/control-structures.rst]: Maybe worth mentioning that it returns false in the `success` part of the return values

File: docs/control-structures.rst [hunk 2]
```diff
@@ -398,17 +398,17 @@ If used properly, analysis tools can evaluate your contract to identify the cond
...
 of an exception instead of "bubbling up".
 
 .. warning::
-    The low-level ``call``, ``delegatecall``, ``callcode`` and ``staticcall`` will return success if the called account is non-existent, as part of the design of EVM. Existence must be checked prior to calling if desired.
+    The low-level ``call``, ``delegatecall`` and ``staticcall`` will return success if the called account is non-existent, as part of the design of EVM. Existence must be checked prior to calling if desired.
```
Comment (leonardoalt) [inline docs/control-structures.rst]: Same here

File: docs/control-structures.rst (hunk 1 above)
Comment (chriseth) [reply docs/control-structures.rst]: Good catch!

**Review by leonardoalt — DISMISSED** [review]

File: docs/control-structures.rst [hunk 3]
```diff
@@ -398,17 +398,17 @@ If used properly, analysis tools can evaluate your contract to identify the cond
...
+    whas deprecated in version 0.4.13 and removed in version 0.5.0.
 
 When exceptions happen in a sub-call, they "bubble up" (i.e. exceptions are rethrown) automatically. Exceptions to this rule are ``send``
-and the low-level functions ``call``, ``delegatecall``, ``callcode`` and ``staticcall`` -- those return ``false`` in case
+and the low-level functions ``call``, ``delegatecall`` and ``staticcall`` -- those return ``false`` as their first return value in case
 of an exception instead of "bubbling up".
 
 .. warning::
-    The low-level ``call``, ``delegatecall``, ``callcode`` and ``staticcall`` will return success if the called account is non-existent, as part of the design of EVM. Existence must be checked prior to calling if desired.
+    The low-level ``call``, ``delegatecall`` and ``staticcall`` will return ``true`` in their first return value if the called account is non-existent, as part of the design of EVM. Existence must be checked prior to calling if desired.
```
Comment (erak) [inline docs/control-structures.rst]: Should be ``low-level functions``.

Comment (erak) [reply docs/control-structures.rst]: Ah, could be changed even more, removing future tense and a typo 😉 

The low-level **functions** ``call``, ``delegatecall`` and ``staticcall`` return ``true`` **as** their first return value if the called account is non-existent, ...

**Review by leonardoalt — DISMISSED** [review]

**Pull request merged**